### ISSUE TITLE
Column name of Longitude preferred to be "Long" instead of "Long_"

### ISSUE THREAD

**Issue by lamjys** [CSSEGISandData/COVID-19#1950]
`time_series_covid19_confirmed_US.csv` is added recently.  Suggest that column name of Longitude preferred to be **"Long"** instead of **"Long_"**, in order to be consistent with other csv data file.

**Comment by loankimrobinson** [comment]
It already had been there a while, so you have to recode again

**Comment by jtene001** [comment]
NO, why would you change it now? Everybody coded around their idiosyncrasies in the beginning, why do you now want us to have to go back and change our code?

**Comment by ksgriffin** [comment]
The data has been this way for a while, it would be a breaking change for anyone who uses the data. Worse yet, with no effective way to reach all those who do use this data. Strongly against any metadata changes.

**Comment by jessehamner** [comment]
Additionally -- especially if one imports data into (say) an *SQL, then `long` is a reserved word (it is a data type) and would require additional workarounds for import scripts. I usually use `lon` and `lat` myself, but `Long_` is not a problem.
